### DISCUSSION TITLE
그림자 문제

### DISCUSSION THREAD

**Discussion by YSHFM** [naverz/zepeto-studio-kor#1049] [5. Q&A]
그림자 연산과 관련하여 질문드립니다.

현재 제가 작업 중인 프로젝트에서 유니티에서는 그림자 연산이 정상적으로 됩니다.

 하지만 모바일 환경에서 플레이어로부터 일점 범위 이내에서만 그림자 연산이 되는 것을 확인했습니다.
모바일 환경이기 때문에 그림자 연산문제를 줄이기 위한 정상적인 반응인건지, 아니면 제가 겪고 있는 문제가 비정상 적인 문제인 지 알고 싶습니다.

**Comment by chlguswls32** [comment]
안녕하세요. YSHFM님 그림자 연산관련하여서 확인해보시면 좋을 방법이있어서 말씀드립니다.

1. 씬의 Directional Light를 클릭한다.
2. Inspector 뷰에 보이는 Light 컴포넌트를 클릭한다.
3. Bias 라는 슬라이드로된 프로퍼티 항목이있습니다. 0(가장작은값) 으로 바꿔보시면 좋을것같습니다.

Unity 에서는 Shadow연산이 무겁기때문에 Shadow를 그리는 범위를 정할수있습니다.
Bias항목을 조절해보시면 될것같습니다. 

스크린샷 남겨주시면 좀더 자세히 볼 수 있을것같아요.
좋은하루되세요.

**Reply by YSHFM** [reply]
제가 스샷을 따로 첨부가 불가능한 프로젝트라서 스샷 첨부는 먼저 어려울 것 같습니다,,,ㅜㅜ 

말씀하신 Bias값도 조정해보았으나 마찬가지로 플레이어로부터 일정 범위 이내에서만 오브젝트들이 그림자 연산을 수행하고 있습니다.
ProjectSettings에서 shadow distance값이 150으로 되어 있어 350정도로도 늘려보았지만 증상은 동일합니다.

**Reply by YSHFM** [reply]
![iOS 이미지 (3)](https://user-images.githubusercontent.com/106574541/180903016-a26e839f-0520-4dc7-b77a-e954507af2da.png)
따로 프로젝트 생성해서 첨부했습니다. 

보시다시피 플레이어에게서 멀리 있는 오브젝트는 그림자가 생성되지 않고 있습니다.
